ui/objects/semantic-ui/js/objects/editform/components/BasicInfo.jsx

```diff
@@ -73,79 +73,72 @@ export const BasicInfo = ({ activeIndex, handleActive, record }) => {
             name="metadata.restorationObject.keywords"
             disabled={false}
           />
-        </div>
-        <div className="vert-div predmety__form__div">
+        </Grid.Column>
+        <Grid.Column>
           <TextAreaField
             name="metadata.restorationObject.description"
             aria-label="Popis"
             fieldPath="metadata.restorationObject.description"
             label={
               <FieldLabel
                 htmlFor="metadata.restorationObject.description"
-                className="predmety__form__div__label"
                 label="Popis"
               ></FieldLabel>
             }
           />
-        </div>
-
-        <div className="vert-div predmety__form__div">
-          <LocalVocabularySelectField
+        </Grid.Column>
+        <Grid.Column>
+          <VocabularyTreeSelectField
             optionsListName="ItemTypes"
             fieldPath="metadata.restorationObject.itemTypes"
             multiple={true}
+            clearable
             placeholder="Vyberte typ předmětu"
+            category={category}
```

Comment:
VocabularyTreeSelectField knows nothing about some kind of "category", even more so, if it is used in another repositiories where theres no concept of "category" at all